app/controlplane/internal/biz/organization.go

```diff
@@ -101,18 +102,14 @@ func (uc *OrganizationUseCase) Delete(ctx context.Context, id string) error {
 		}
 	}
 
-	// Delete the associated repository
-	// Currently there is only one repository per organization
```

Comment:
this is a bug we had that affected user offboarding. We only deleted the main cas backend instead all of them.